pkg/executables/helm.go

```diff
@@ -21,6 +21,8 @@ type Helm struct {
 	registryMirror *registrymirror.RegistryMirror
 	env            map[string]string
 	insecure       bool
+	// skipCRDs passes the --skip-crds flag to the helm executable.
+	skipCRDs bool
```

Comment:
what does this do?
By building the helm client with this, all consumers will use this flag (there is only one instance of the helm client per factory)

Reply:
It does what it says on the tin... If the helm chart contains CRDs, they'll be skipped (i.e. not installed on the cluster).

Let me see what I can manage. It'd probably mean moving it as a flag to `Install`.

Reply:
Yeah, moving it to `InstallChart` makes the most sense to me. That's where we set a few other run-time flags, like `kubeconfig`.